tests/app/dao/test_services_dao.py

```diff
@@ -874,6 +874,24 @@ def test_fetch_stats_for_today_only_includes_today(notify_db_session):
     assert stats['created'] == 1
 
 
+def test_fetch_stats_for_today_only_includes_today_in_bst(notify_db_session):
+    template = create_template(service=create_service())
+    # two created email, one failed email, and one created sms with freeze_time('2001-03-27T23:59:00'):
+    # just before midnight yesterday in UTC create_notification(template=template, to_field='1', status='delivered')
+    with freeze_time('2001-03-27T23:01:00'):
+        # just after midnight yesterday in UTC
+        create_notification(template=template, to_field='2', status='failed')
+    with freeze_time('2001-03-28T12:00:00'):
+        # right_now, we have entered BST at this point but had not for the previous two notifications
```

Comment:
This references previous two notifications but there is only one previous notification?